File: Tabuada.py
n = int(input('Digite qual a tubuada que deseja ->'))                   #Entrada e conversao do numero que iremos fazer a tabuada  /  Entry and conversion of the number that we will make the multiplication table
nvzs = int(input('Digite ate que numero quer que va a tabuada ->'))        #informando ate que numero a tabuada vai  /  informing the number of times tables

vzs = 0              #atribuindo 0 na variavel de controle  /  assigning 0 to the control variable

for vzs in range (nvzs+1):          #loop for
    print (n,'X',vzs,'=', n*vzs)       #imprimindo texto e dados e o resultado  /  printing text and data and the result

File: tabuada.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
	int x, z, rep = 0;
    int y = 1;
do{
	printf("Digite qual tabuada voce quer\n");
	scanf("%i", &x);
	printf("Digite ate que numero voce quer que va a tabuda\n");
	scanf("%i", &z);

	system ("cls");
	printf ("\t\tTABUADA DO %i\n", x);

            for ( ; y <= z; ++y ){
                printf("%ix%i=%i\n", x, y, x*y);
            }

		printf("\t\tQuer ver outra tabuada?\n \t1-Sim\n \t2-Nao\n");
		scanf("%i", &rep);

			system ("cls");
			y = 1;

	}while(rep==1);

	return 0;
}
